salt/metalk8s/repo/installed.sls

```diff
@@ -70,9 +19,16 @@ Install repositories manifest:
     - name: /etc/kubernetes/manifests/repositories.yaml
     - source: salt://{{ slspath }}/files/repositories-manifest.yaml.j2
     - config_files:
-      - {{ nginx_default_conf }}
-      - {{ nginx_registry_path }}
-      - {{ nginx_registry_config_path }}
+      - {{ salt.file.join(repo.config.directory, repo.config.default) }}
```

Comment:
minor: could be a jinja for loop over `repo` keys? That might be ugly and too long though.

Reply:
No I think we don't want to do that because it's something merged with the pillar so we don't want to take all of them